src/components/Footer.js

```diff
@@ -1,20 +1,30 @@
 import { NavLink } from "react-router-dom";
 import "./Footer.css";
-import circlePlus from "../images/circle-plus-fill.svg";
-import timer from "../images/timer.svg";
-import heart from "../images/heart.svg";
+import { IoTimerOutline, IoAddCircle, IoHeart } from "react-icons/io5";
 
 export default function Footer() {
   return (
     <nav className="NavBar">
-      <NavLink to="/create">
-        <img className="NavIcon AddIcon" src={circlePlus} alt="Create icon" />
+      <NavLink
+        to="/create"
+        className="NavIcon"
+        activeClassName="NavIcon--active"
+      >
+        <IoAddCircle alt="Create icon" />
       </NavLink>
-      <NavLink to="/myList">
-        <img className="NavIcon" src={timer} alt="My List icon" />
+      <NavLink
+        to="/myList"
+        className="NavIcon"
+        activeClassName="NavIcon--active"
+      >
+        <IoTimerOutline alt="My List icon" />
       </NavLink>
-      <NavLink to="/favourites">
-        <img className="NavIcon" src={heart} alt="My Favourites icon" />
+      <NavLink
+        to="/favourites"
+        activeClassName="NavIcon--active"
+        className={"NavIcon"}
```

Comment:
What are the curly braces here but not set with other classes?

Reply:
I was trying out things and forgot that one. Tks!